integration-testing/build.gradle

```
@@ -92,8 +92,8 @@ task coreAgentTest(type: Test) {
 }
 
 dependencies {
-    testCompile "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
-    testCompile 'junit:junit:4.12'
+    testImplementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    testImplementation 'junit:junit:4.12'
     npmTestCompile 'org.apache.commons:commons-compress:1.18'
```

```suggestion
    npmTestImplementation 'org.apache.commons:commons-compress:1.18'
```
I think it's okay to also replace this (and dependencies below)

---

I was not sure, as it seems custom configuration here

---

There are source sets matching these configurations (https://github.com/Kotlin/kotlinx.coroutines/blob/6af3161dbc0972b0b6e68372b9abdae722b02a4b/integration-testing/build.gradle#L16)